### ISSUE TITLE
Cannot access revisions

### ISSUE THREAD

**Issue by condor616** [droptica/droopler#518]
When trying to access revisions for a particular node, I get the following error: 

```
ErrorMessage: Call to a member function getType() on string in droopler_theme_is_special_route() (line 399 of /var/www/html/web/profiles/contrib/droopler/themes/custom/droopler_theme/droopler_theme.theme)
```

**Comment by rafal-sereda-droptica** [comment]
Thanks for reporting, we'll take a look and try to fix it.

**Comment by condor616** [comment]
Thank you. Please let me know if you need additional details, in case you’re not able to reproduce on a default installation.

**Comment by pawel-holik-droptica** [comment]
Hey @condor616 
looks like we found and fixed that issue. It should be included in the nearest release.

**Comment by rafal-sereda-droptica** [comment]
Hello, this is fixed in the 2.1-rc2 release.
